.github/workflows/build-sample-apps.yml

```diff
@@ -0,0 +1,70 @@
+name: Build sample apps 
```

Comment:
Cool fact, this CI workflow is the base structure required to add multiple sample apps into our SDK code base! 

Great to get part of this work done 😄